### ISSUE TITLE
Support Facebook Pages using faceprox

### ISSUE THREAD

**Issue by kimadactyl** [geeksforsocialchange/PlaceCal#1365]
## User story

As a partner or coordinator, I want to list my events on PlaceCal using Facebook, in order that I don't have to do anything extra to update it.

## Acceptance criteria

- [ ] If I add "https://facebook.com/oldabbeytaphouse" to the calendar URL field, my events get imported
- [ ] If the proxy server goes down, we get some kind of notification 
- [ ] Ensure below calendars are working
- [ ] Make a new ticket for supporting Groups (this is waiting on [faceprox features](https://github.com/geeksforsocialchange/faceprox/issues/6))

```
oldabbeytaphouse
HulmeCommunityGardenCentre
regentcentre
bavscharity
rockdoverising
CCPbh23
CommunityArtsNorthWest
Olivepathway
grrrlzinefair
```

## Implementation notes & questions

We now have [faceprox](https://github.com/geeksforsocialchange/faceprox) to redirect the requests to.

~#1351 is possibly a prerequisite for doing this cleanly~ - done

Faceprox server should be up at http://a.faceloader.link/ soonish

## Implementation plan

_To be written by the developer_

- [ ] Faceprox URL should probably be set with an environment variable

**Comment by kimadactyl** [comment]
@nachimir @GeorgieMcVicar feel free to add more facebook pages to this thread and we will add them in a batch when this is implemented

**Comment by katjam** [comment]
Dream yard
